common/conf.hpp

```diff
@@ -233,6 +235,8 @@ class MenderConfig : public cfg_parser::MenderConfigFromFile {
 public:
 	Paths paths {};
 
+	http::ClientConfig http_client_config;
```

Comment:
Wouldn't it be better to add `http::ClientConfig GetClientConfig()`? What if something changes some value the config is based on **after** `ProcessCmdlineArgs()`?

Reply:
I suppose that's good future proofing, yes. I'll switch to that.